### PR TITLE
Add a Quassel IRC Helm Chart

### PR DESCRIPTION
**Pull request by kiall**
Add a Quassel IRC Helm Chart

Quassel IRC is a modern, cross-platform, distributed IRC client, meaning that one (or multiple) client(s) can attach to and detach from a central core.

Quassel Homepage: https://quassel-irc.org/

### CODE REVIEW

**Comment by kiall** [comment]
Assigning seanknox as requested by the bot :)

File: incubator/quassel/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+appVersion: 0.12.4
+description: Quassel IRC is a modern, cross-platform, distributed IRC client.
+name: quassel
+version: 0.2.0
+icon: https://avatars3.githubusercontent.com/u/2965670
+home: https://quassel-irc.org/
+maintainers:
+- name: Kiall Mac Innes
```
Comment (unguiculus) [inline incubator/quassel/Chart.yaml]: Please use Github username.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR. Looks really good. I'd suggest you move it to stable directly.

File: incubator/quassel/Chart.yaml (hunk 1 above)
Comment (kiall) [reply incubator/quassel/Chart.yaml]: Done

**Comment by kiall** [comment]
> Thanks for the PR. Looks really good. I'd suggest you move it to stable directly.

Done.

EDIT: Let me update the readme references. Derp.

**Comment by kiall** [comment]
> EDIT: Let me update the readme references. Derp.

Done.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**